qhub/schema.py

```diff
@@ -54,6 +54,7 @@ class Config:
 class CICD(Base):
     type: CiEnum
     branch: str
+    commit_render: bool
```

Comment:
Make optional with default of true? Open to what we would like the default value to be.

Reply:
I agree it makes sense to make it optional :)